src/Command/DataFlow/DataFlowProduceCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of itk-dev/datatidy.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Command\DataFlow;
+
+use App\Entity\DataFlow;
+use App\Entity\DataFlowJob;
+use App\Event\DataFlowJobCreatedEvent;
+use App\Event\DataFlowJobQueuedEvent;
+use App\Message\RunDataFlowJobMessage;
+use App\Repository\DataFlowRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class DataFlowProduceCommand extends Command
```

Comment:
For consistency this class should be called `DataFlowProduceJobsCommand`.

Reply:
Done.